frontend/src/metabase/admin/settings/components/SettingsPages/PublicSharingSettingsPage.tsx

```diff
@@ -0,0 +1,48 @@
+import { t } from "ttag";
+
+import { useSetting } from "metabase/common/hooks";
+import { Stack, Text } from "metabase/ui";
+
+import { AdminSettingInput } from "../widgets/AdminSettingInput";
+import {
+  PublicLinksActionListing,
+  PublicLinksDashboardListing,
+  PublicLinksQuestionListing,
+} from "../widgets/PublicLinksListing";
+
+export function PublicSharingSettingsPage() {
+  const publicSharingEnabled = useSetting("enable-public-sharing");
+  return (
+    <Stack gap="xl" p="2rem 2rem 2rem 1rem">
+      <AdminSettingInput
+        name="enable-public-sharing"
+        title={t`Enable Public Sharing`}
+        inputType="boolean"
+      />
+      {publicSharingEnabled && (
+        <Stack gap="xl">
+          <Stack gap="sm">
+            <Text c="text-medium" fw="bold" tt="uppercase" display="block">
+              {t`Shared Dashboards`}
+            </Text>
+
+            <PublicLinksDashboardListing />
+          </Stack>
+          <Stack gap="sm">
+            <Text c="text-medium" fw="bold" tt="uppercase" display="block">
```

Comment:
you probably just want to
```
import { SettingTitle } from "../SettingHeader";
```

Reply:
The reason I didn't is because label doesn't feel right for these headings because they don't belong to an individual input ie `htmlFor` doesn't really make sense IMO

Reply:
you could make htmlFor/id optional?

Reply:
If it was just for the benefit of this page I would vote against loosening the types. But if it's common I guess we could. 

Fundamentally I believe this header is a different component than the `SettingInput`, it's just that the original design displays them similarly
- `SettingHeader`: Header for a setting input
- `PublicListingHeader`: Header for a setting section

When the new designs come in, this will be rectified so the SettingSectionHeader looks visibly different form the SettingHeader
<img width="500" alt="Screenshot 2025-05-06 at 1 39 28 PM" src="https://github.com/user-attachments/assets/50ca9862-f04f-44f3-8dc2-de8efb593057" />